test/test_email_notifier.py

```diff
@@ -0,0 +1,166 @@
+from unittest.mock import MagicMock, mock_open, patch
+
+import pytest
+
+from wadas.domain.detection_event import DetectionEvent
+from wadas.domain.email_notifier import EmailNotifier
+
+
+@pytest.fixture
+def email_notifier():
+    with patch("wadas.domain.email_notifier.keyring.get_credential") as mock_cred:
+        mock_credential = MagicMock()
+        mock_credential.username = "sender@example.com"
+        mock_credential.password = "fake_password"
+        mock_cred.return_value = mock_credential
+
+        return EmailNotifier(
+            sender_email="sender@example.com",
+            smtp_hostname="smtp.example.com",
+            smtp_port=465,
+            recipients_email=["recipient1@example.com", "recipient2@example.com"],
+            enabled=True,
+        )
```

Comment:
Same here. It's more or less (there are a few changes, in some case) the same repeated below on other tests.